snoop/data/migrations/0041_auto_20210215_1046.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 3.1.4 on 2021-02-15 10:46
+
+from django.db import migrations
+
+entity_type_list = ['ORG', 'PER', 'LOC', 'CARDINAL', 'DATE', 'EVENT', 'FAC', 'GPE',
+                    'LANGUAGE', 'MONEY', 'NORP', 'ORDINAL', 'PERCENT', 'PERSON',
+                    'PRODUCT', 'QUANTITY', 'TIME', 'WORK_OF_ART']
+
+
+def create_entity_types(apps, schema_editor):
+    Type_model = apps.get_model('data', 'EntityType')
+    db_alias = schema_editor.connection.alias
+    Type_model.objects.using(db_alias)
+    Type_model.objects.using(db_alias).bulk_create([
+        Type_model(type=entity)
+        for entity in entity_type_list
+    ])
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('data', '0040_auto_20210215_1045'),
+    ]
+
+    operations = [
+        migrations.RunPython(create_entity_types),
```

Comment:
Set NoOP for reverse of migration (see other RunPython migrations around here), so we can reverse it.